odml/tools/odmlparser.py

```diff
@@ -240,24 +240,40 @@ def fromFile(self, file):
                 file.close()
             return self.to_odml()
 
+        # TODO discuss whether local import is appropriate here
+        elif self.parser == 'RDF':
+            if not doc_format:
+                raise KeyError("Format of the rdf file was not specified")
+            from odml.tools.rdf_converter import RDFReader
```

Comment:
I'd always put import statements at the top unless its a circular dependency.

Reply:
Circular dependency here.